fix(rebranding): Update all static text references to CodeReady Toolchain

nimishamukherjee opened the pull request:
Update all static text references to CodeReady Toolchain on OSIO - [OSIO-1180](https://openshift.io/openshiftio/Openshift_io/plan/detail/1180)

Attaching screenshots for some of the code changes:

Feature Opt In
![featureoptin](https://user-images.githubusercontent.com/1892722/49517562-27062e00-f862-11e8-8bf1-12f7fa3d0465.png)

Context text during space creation
![context](https://user-images.githubusercontent.com/1892722/49517609-3be2c180-f862-11e8-8d4f-33f5bce9adce.png)

Context text during app creation
![app](https://user-images.githubusercontent.com/1892722/49517652-56b53600-f862-11e8-81a6-b33c6db5eba0.png)


Remove codebase modal
![removecodebase](https://user-images.githubusercontent.com/1892722/49517629-4c933780-f862-11e8-9be0-892e13f00112.png)




Reference: https://github.com/openshiftio/openshift.io/issues/4364

christianvogt commented:
@nimishamukherjee I do not see the changes related to the app creation wizard. Also note that the screenshot shows "ToolChain" instead of "Toolchain".

Could you also fix the text from "When I import a codebase?" to the correct title of "When should I import a codebase?".
This is in the same file as the one missing from the PR.

nimishamukherjee commented:
@christianvogt 
> ToolChain" instead of "Toolchain".

fixed in 035e4a1485f0d4782a13ac3d41c41c121a03d4b1

> "When I import a codebase?" to the correct title of "When should I import a codebase?"

fixed in 2cf1daa01694c2293ec3a7a2c0d7ebf368e6dec0

fabric8cd commented:
:tada: This PR is included in version 3.0.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/fabric8-ui)
- [GitHub release](https://github.com/fabric8-ui/fabric8-ui/releases/tag/v3.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
